### ISSUE TITLE
sonarr/nzbdrone start-stop-status unable to start application

### ISSUE THREAD

**Issue by marcodings** [SynoCommunity/spksrc#2916]
_For new Package Requests, see the guidelines_  

### Setup
_Package Name:_sonarr
_Package Version:_20170106-9

_NAS Model:_ds1513+
_NAS Architecture:_
_DSM version:_

### Expected behavior
running as root:
    /var/packages/nzbdrone/scripts/start-stop-status stop
    /var/packages/nzbdrone/scripts/start-stop-status start
i would expect sonarr/nzbdrone to be running

### Actual behavior
It is will not run when trying to start from cli when logged in as root

Starting via the packagemanager entry for sonar works fine. See entry under "Other logs"
For debugging purposes i added "set -x" to the start-stop-status start
The commands run as root and the ones "through the package manger" did have a different PATH which i corrected, to no avail. At this point the output from the start and stop commands themselves it equal in both cli and package manager

When using the "run package" from the package manager it actually performs a stop and then a start ( see log ). However in - between it does 2 other things

    2017/09/18 18:05:40     start nzbdrone 20170106-9 End start-stop-status stop ret=[0]

    2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin unload apparmor
    2017/09/18 18:05:40     start nzbdrone 20170106-9 End unload apparmor ret=[0]
    2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin pre-load apparmor
    2017/09/18 18:05:40     start nzbdrone 20170106-9 End pre-load apparmor ret=[0]

    2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin start-stop-status start

This leads me to believe/hypothesize that the commands to do
- unload apparmor
- pre-load apparmor
are somehow crucial for subsequent successfull "running"/starting

But i have no clue as where to find them / how to initiate them.

Then again it could be something different but this is how far i got ;)

### Steps to reproduce
_1._/var/packages/nzbdrone/scripts/start-stop-status stop
_2._/var/packages/nzbdrone/scripts/start-stop-status start
_3._/var/packages/nzbdrone/scripts/start-stop-status status

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```
Insert the package log here
cat /usr/local/nzbdrone/var/install.log
      || Installing package version="20170106-9" - Thu Aug 24 13:39:34 CEST 2017 ||
      || Package Install Completed - Thu Aug 24 13:39:37 CEST 2017 ||

#### Other logs
_E.g. `/var/log/messages` or `/var/log/synopkg.log`_
```
Insert log here
tail -n 35  /var/log/synopkg.log

     + COMMAND='env PATH=/usr/local/mono/bin:/usr/local/nzbdrone/bin:/sbin:/bin:/usr/sbin:/usr/bin:   /usr/syno/sbin:/usr/syno/bin:/usr/local/sbin:/usr/local/bin LD_LIBRARY_PATH=/usr/local/nzbdrone/lib /usr/local/mono/bin/mono -- --debug /usr/local/nzbdrone/share/NzbDrone/NzbDrone.exe'
     + case $1 in
     + daemon_status
     + start-stop-daemon -K -q -t -u nzbdrone -p /usr/local/nzbdrone/var/.config/NzbDrone/nzbdrone.pid   
     + echo Sonarr is not running
     Sonarr is not running
     2017/09/18 18:05:40     start nzbdrone 20170106-9 End start-stop-status stop ret=[0]

     2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin unload apparmor
     2017/09/18 18:05:40     start nzbdrone 20170106-9 End unload apparmor ret=[0]
     2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin pre-load apparmor
     2017/09/18 18:05:40     start nzbdrone 20170106-9 End pre-load apparmor ret=[0]

    2017/09/18 18:05:40     start nzbdrone 20170106-9 Begin start-stop-status start
    + INSTALL_DIR=/usr/local/nzbdrone
    ....

**Comment by mathieubernou** [comment]
I am interested in this topic because I have exactly the same problem. Any progress?

**Comment by Safihre** [comment]
Test packages for DSM6.1 and various architectures are available here #3138. 
They also include a new way of handling permissions and should fix existing issues.
We hope to publish the packages to the official repo soon and appreciate all testing and reports on the functioning of these packages.
